code/modules/mob/living/carbon/life.dm

```diff
@@ -35,6 +35,17 @@
 		stop_sound_channel(CHANNEL_HEARTBEAT)
 		rot()
 
+	if(stat != DEAD)
+		if(InCritical())
+			critical_ticks++

```

Comment:
This is a pretty bad way to keep track of how much time has passed, especially in a process proc

Reply:
How should I do it, then?

Reply:
process ticks almost always have an incremental variable. Breath code especially so.

Reply:
How ever it would be more effective to put a var that gets the world time the moment a mob enters a critical state, then checks for it `world.time + 5 MINUTES` before doing the heal ticks.